tests/zap_helper_test.py

```diff
@@ -8,7 +8,7 @@
 import subprocess
 import unittest
 import urllib2
-
+import io
```

Comment:
Minor: As a convention in this repo, we separate core imports and those from installed packages and local modules by a new line, so can you add a new line after the `io` import?